cert-manager-multi-arch.md

```diff
@@ -0,0 +1,728 @@
+```shell
+$ curl -O https://mirror.openshift.com/pub/openshift-v4/ppc64le/clients/ocp/stable/openshift-install-linux-amd64.tar.gz
+```
+```shell
+$ tar -xvf ./openshift-install-linux-amd64.tar.gz && rm ./openshift-install-linux-amd64.tar.gz
+```
+
+
+
+4. Download and extract `oc` binary
+```shell
+$ curl -O https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz
+```
+```shell
+$ tar -xvf ./openshift-client-linux.tar.gz && rm ./openshift-client-linux.tar.gz
+```
+
+
+
+5. Once all downloads are complete, ensure that the `assets` directory contains four binaries. You may also use the `--help` command to verify compatibility with your system for each binary.
+
+```shell
+$ ls
+ccoctl  kubectl  oc  openshift-install
+$ ./ccoctl --help
+....
+$ ./openshift-install --help
+...
+```
+
+### Create cluster installation manifests
+
+**Prerequisites**
+
+* [SSH key pair for cluster node access](https://docs.openshift.com/container-platform/4.14/installing/installing_ibm_powervs/installing-ibm-power-vs-customizations.html#ssh-agent-using_installing-ibm-power-vs-customizations)
+* [Red Hat OpenShift Pull secret](https://console.redhat.com/openshift/install/pull-secret)
+
+
+1. Create `cluster-assets` directory inside `assets` directory to store the files required for cluster installation.
+
+```shell
+$ mkdir cluster-assets
+```
+
+2. Generate the configuration file
+
+```shell
+$ ./openshift-install create install-config --dir ./cluster-assets
+```
+
+Upon executing this command, provide the following information when prompted
+
+```shell
+./openshift-install create install-config --dir ./cluster-assets
+
+? SSH Public Key /home/sandbox/.ssh/id_rsa.pub                           <your ssh pub key>
+? Platform powervs                                                       <select powervs>
+? IBM Cloud User ID abc@example.com                                      <your ibm cloud id>
+? Region dal                                                             <power vs workspace region>
+? Zone dal10                                                             <power vs workspace zone : echo $DATACENTER>
+? Service Instance abcd123-efgh456-qwer-sdf-864gsj123                    <power vs workspace guid : echo $WORKSPACE_ID>
+? Resource Group sandbox-rg                                              <power vs workspace resource group : echo $RESOURCE_GROUP>
+? Base Domain example.com                                                <the base domain to deploy the cluster to>
+? Cluster Name sandbox-cluster                                           <your cluster name>
+? Pull Secret *********                                                  <your openshift pull secret> 
+```
+
+After execution, the installation program will store the configuration in 
+`./cluster-assets/install-config.yaml` file.
+
+You can view the contents by running `cat ./cluster-assets/install-config.yaml`, which will resemble like:
```

Comment:
I would prefer adding a version number here, like `at the time of writing this we are using OpenShift Container Platfor v4.14` or something similar. Putting it here near install-config may or may not make sense depending upon how things change in future, but a good add about the OCP version number somewhere highlighted in text.